src/field/external_gps.md

```diff
@@ -1,44 +1,66 @@
-  -On older Android versions, the developer settings can be made visible under the **Android Settings** > **Applications** > **Development**.
-- Once you have the developers option enabled, go to **System** > **Advanced** > **Developer options** > **Select mock location app** and select **Bluetooth GPS**
+To enable **Allow mock locations**:
+- In Android device, enable **Developer options**, which are hidden by default:
+  - On newer Android versions, go to **Android Settings** > **About phone** and scroll down to find the **Build number**. Tap the build number 7 times. A message appears with a count-down until you **become a developer**.
+  - On older Android versions, the developer settings can be made visible under the **Android Settings** > **Applications** > **Development**.
+- Once you have the developers option enabled, go to **System** > **Advanced** > **Developer options** > **Select mock location app** and select your GPS app.
 
-![Mcok location](./external_gps_android_mock.png)
+The mock location will be used by Input automatically as if received from the internal receiver.
 
-Now, pair your external GPS device to your mobile/tablet through Bluetooth. Then open the **Bluetooth GPS** app and from the **Select paired GPS device and connect** drop-down menu, select your external GPS. Make sure you have also selected the box for **Enable Mock GPS Provider**. Select **Connect**. You can now open Input and it should be using the external GPS device.
+## External GPS in iOS
 
-![GPS Bluetooth](./external_gps_android_bluetooth.png)
+External GPS can be connected to your iOS device via Bluetooth. Depending on the GPS manufacturer, there may be a custom app, which then provides position to your iOS device. Input sees this as an internal GPS receiver automatically, without additional configuration.
 
+![GPS iOS](./external-gps-ios.png)
 
-## iOS
-To use your external GPS device with iPhone or iPad, you need to pair the device via Bluetooth. Once the device is paired, the signal from external GPS will be used as the primary source.
+## External GPS troubleshooting
+If you encounter issues with external GPS in Input, make sure that:
+- your Bluetooth is turned on
+- GPS is paired with your mobile device
+- your GPS receiver is turned on, has battery and is sending data
+- remove the device in Input, and add it again from Input
 
-::: warning
-Users have reported that they have to switch on Aeroplane mode to be able to use the external GPS device.
-:::
+## List of known supported GPS devices
+External GPS functionality depends on the manufacturer and on the specific model of your GPS device. These devices are currently known to work well:
+
+- **Emlid Reach RS+** - directly via Bluetooth connection, has an internal NTRIP client to receive corrections. Android only.
```

Comment:
We can also have a markdown table , and this text below table as extra details.

- Manufacturer | Model | Android | iOS | 
- XX.                  | YYYY  | yes/no/unknown | yes/no/unknown |